pkg/ndn-riot/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=ndn-riot
+PKG_URL=https://github.com/astralien3000/ndn-riot
```

Comment:
What is the reason not to take the official repo's url?

Reply:
New errors appeared since the last time I tried the code. I will need to do a pull request to the official repos, but for the moment, I try with mine. Of course this should not be merged for the moment.

Reply:
Just in case, I can quickly merge pull requests in named-data-iot/ndn-riot repo